pkg/volume/flexvolume/flexvolume_util.go

```diff
@@ -20,16 +20,27 @@ import (
 	"encoding/json"
 	"errors"
 	"fmt"
+	"time"
 
 	"github.com/golang/glog"
 
+	"k8s.io/kubernetes/pkg/api"
 	"k8s.io/kubernetes/pkg/util/exec"
+	"k8s.io/kubernetes/pkg/volume"
 )
 
 const (
-	initCmd    = "init"
-	attachCmd  = "attach"
-	detachCmd  = "detach"
+	initCmd = "init"
+
+	attachCmd             = "attach"
+	waitForAttachCmd      = "waitforattach"
+	getDeviceMountPathCmd = "getdevicemountpath"
+	mountDeviceCmd        = "mountdevice"
+
+	detachCmd        = "detach"
+	waitForDetachCmd = "waitfordetach"
+	unmountDeviceCmd = "unmountdevice"
+
 	mountCmd   = "mount"
```

Comment:
We can still keep the old mountCmd calls. No need for now to do "mountDeviceCmd", unless there is a usecase.